### PR TITLE
Aligned CTA button on login flow

### LINKED ISSUE #825: Align CTA button on login flow.
To avoid flickering effect:

CTA Moving

Should be more aligned:

image

### PR DESCRIPTION
**Pull request by surakin**
Adjusted the login flow buttons a little bit
They were slightly smaller on the onboarding page so I used that everywhere

 
## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Technical
- [x] Other : cosmetic

## Content

Aligned the button on the login flow as requested

## Motivation and context

Fixes #825 

## Screenshots / GIFs
Screenshot_20240111_181820
Screenshot_20240111_181832
Screenshot_20240111_181842
Screenshot_20240111_181903

## Tests

Play around the initial flow

## Tested devices

- [ ] Physical
- [x] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [x] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [x] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR

### CODE REVIEW

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.lockscreen.impl.setup.biometric_SetupBiometricView_null_SetupBiometricView-Day-2_3_null_0,NEXUS_5,1.0,en].png
Comment (frebib) [inline tests/uitests/src/test/snapshots/images/ui_S_t[f.lockscreen.impl.setup.biometric_SetupBiometricView_null_SetupBiometricView-Day-2_3_null_0,NEXUS_5,1.0,en].png:1]: Text doesn't fit the button here now

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.onboarding.impl_OnBoardingScreen_null_OnBoardingScreen-Night-0_2_null_3,NEXUS_5,1.0,en].png
Comment (frebib) [inline tests/uitests/src/test/snapshots/images/ui_S_t[f.onboarding.impl_OnBoardingScreen_null_OnBoardingScreen-Night-0_2_null_3,NEXUS_5,1.0,en].png:1]: Spacing is off here too

**Comment by surakin** [comment]
Should I try to fix these? It's not like they are part of any enabled functionality yet :thinking:

**Comment by jmartinesp** [comment]
Thanks for the PR! IMO, the new alignment and size of the buttons looks better, somehow making them a bit smaller than the rest of the content makes them stand out more. However, any change in designs like this must be approved by our design team, since it's part of our effort to keep designs consistent across platforms. What do you think @amshakal ?

Also, I think @bmarty already did some work in this area, so maybe he knows better than me what decision we took about this (if we did).

On the other hand, about the 'broken' screenshots: I'd say we can make an exception for the onboarding screen and not follow this positioning of the 'Continue' button since it's the very first one if I'm not mistaken. As for the issue with the 'allow biometric' button text being cut, maybe we could change it from 'Allow biometric authentication' to 'Allow biometric unlock' or 'Use biometric authentication' so it fits.

Anyway, let's see what the design team thinks about this changes before discussing any changes.

**Comment by jmartinesp** [comment]
@surakin sorry for the delay, but our design team is on holiday until next week 😓

**Comment by surakin** [comment]
No rush :) This can wait

**Comment by amshakal** [comment]
Heya! Thanks for contributing. :)
The buttons position and width looks good. I am happy to approve these suggestions from the design side.

**Comment by jmartinesp** [comment]
Thanks @amshakal ! @surakin can you fix the conflicts?

**Comment by surakin** [comment]
That should do!

**Pull request merged**